cleanlab_studio/utils/databricks_utils.py

```diff
@@ -0,0 +1,135 @@
+import os
+import zipfile
+from datetime import datetime
+from pathlib import Path
+
+
+def dbfs_to_posix_path(dbfs_path: str) -> str:
+    """
+    Converts a DBFS path to a POSIX path.
+
+    Args:
+        dbfs_path: The DBFS path to convert.
+
+    Returns:
+        The POSIX path.
+    """
+    return f"/dbfs{dbfs_path[5:]}"
+
+
+def get_databricks_imageset_df_image_col(df) -> str:
```

Comment:
Should add typing to the `df` input here (and other places in `databricks_utils` and `snowflake_utils` where it's missing